src/main/java/org/apache/cassandra/distributed/api/NodeToolResult.java

```diff
@@ -134,6 +144,30 @@ public Asserts notificationContains(ProgressEventType type, String msg)
             return this; // unreachable
         }
 
+        public Asserts capturedConsoleOutContains(String substring)
+        {
+            return capturedConsoleOutputContains(substring, true);
+        }
+
+        public Asserts capturedConsoleErrContains(String substring)
+        {
+            return capturedConsoleOutputContains(substring, false);
+        }
+
+        private Asserts capturedConsoleOutputContains(String substring, boolean isOut)
+        {
+            String name = isOut ? "capturedConsoleOut" : "capturedConsoleErr";
+            String output = isOut ? capturedConsoleOut : capturedConsoleErr;
+            AssertUtils.assertNotNull(name + " not defined", output);
+            AssertUtils.assertFalse("Found no " + name, output.isEmpty());
```

Comment:
do we care about that, `""` does not contain `"match"` so the fail should be fine